StratisCore.UI/src/app/setup/create/show-mnemonic/show-mnemonic.component.ts

```diff
@@ -3,20 +3,23 @@ import { Router, ActivatedRoute } from '@angular/router';
 import { Subscription } from 'rxjs';
 
 import { WalletCreation } from '@shared/models/wallet-creation';
+import { GlobalService } from '@shared/services/global.service';
 
 @Component({
   selector: 'app-show-mnemonic',
   templateUrl: './show-mnemonic.component.html',
   styleUrls: ['./show-mnemonic.component.css']
 })
 export class ShowMnemonicComponent implements OnInit, OnDestroy {
-  constructor(private route: ActivatedRoute, private router: Router) { }
+  constructor(private route: ActivatedRoute, private router: Router, private globalService: GlobalService) { }
   private mnemonic: string;
   private subscription: Subscription;
   private newWallet: WalletCreation;
   public mnemonicArray: string[];
+  public sidechainEnabled: Boolean;
```

Comment:
I think the best practive is not to use Boolean (the one with capital B), because
```
var falseObj = new Boolean(false);
if (falseObj) {
    console.log('our intent may not be to get here');
}
```